Rossum/README.md

```diff
@@ -0,0 +1,37 @@
+# What are Box Skills?
```

Comment:
Awesome! could we attach some screenshots of what files and the cards next to then looks like in the README? That will be really helpful for anyone looking through skills to pick up something they want to use.